File: StudyApp/StudyApp.API/Domain/Entities/Blogs.cs

namespace StudyApp.API.Domain.Entities
{
    public class Blogs : AuditEntity
    {
        public string Title { get; set; }
        public string ShortDescription { get; set; }

        public string Content { get; set; }
        public string ImageUrl { get; set; }

       
    }
}

File: study-app-frontend/src/types/blogs.ts
export interface BlogsModel{
    id: number; // From AuditEntity
    title: string;
    shortDescription: string; 
    content:string;
    imageUrl?:string;
    createdAt?:Date
}

export interface Blogsdetail{
    title:string;
    content:string;
}